Apps/W1/Shopify/test/Companies/ShpfyCompanyLocationsTest.Codeunit.al

```diff
@@ -0,0 +1,142 @@
+codeunit 139539 "Shpfy Company Locations Test"
+{
+    Subtype = Test;
+    TestPermissions = Disabled;
+    TestHttpRequestPolicy = BlockOutboundRequests;
+
+    var
+        ShpfyCompanyLocation: Record "Shpfy Company Location";
+        Customer: Record Customer;
+        IsInitialized: Boolean;
+        ResponseResourceUrl: Text;
+
+    trigger OnRun()
+    begin
+        IsInitialized := false;
+    end;
+
+    [Test]
+    [HandlerFunctions('HttpSubmitHandler')]
+    procedure TestCreateCompanyLocationSuccess()
+    var
+        ShpfyCompany: Record "Shpfy Company";
+        ShpfyCompanyAPI: Codeunit "Shpfy Company API";
+        ShpfyCompanies: TestPage "Shpfy Companies";
+    begin
+        // [Given] A valid customer and company location setup
+        this.Initialize();
+        ShpfyCompany.GetBySystemId(this.ShpfyCompanyLocation."Company SystemId");
+        // [When] CreateCompanyLocation is called
+        ShpfyCompanyAPI.CreateCompanyLocation(this.ShpfyCompanyLocation, this.Customer);
+
+        // [Then] Company location should be created successfully
+        this.ShpfyCompanyLocation.SetRange("Customer Id", this.Customer.SystemId);
+        this.ShpfyCompanyLocation.FindFirst();
+        ShpfyCompanies.OpenEdit();
+        ShpfyCompanies.GoToRecord(ShpfyCompany);
+        ShpfyCompanies.Locations.GoToRecord(this.ShpfyCompanyLocation);
+    end;
+
+    [Test]
+    procedure TestCreateCompanyLocationCustomerAlreadyExportedAsCompany()
+    var
+        ShpfyCompany: Record "Shpfy Company";
+        ShpfySkippedRecord: Record "Shpfy Skipped Record";
+        LibraryAssert: Codeunit "Library Assert";
+        ShpfyCompanyAPI: Codeunit "Shpfy Company API";
+    begin
+        // [Given] Customer already exported as a company
+        this.Initialize();
+        ShpfyCompany.GetBySystemId(this.ShpfyCompanyLocation."Company SystemId");
+        ShpfyCompany."Customer SystemId" := Customer.SystemId;
+        ShpfyCompany.Modify(true);
+
+        // [When] CreateCompanyLocation is called
+        ShpfyCompanyAPI.CreateCompanyLocation(this.ShpfyCompanyLocation, this.Customer);
```

Comment:
This doesn't compile. The parameter list for CreateCompanyLocation only contains "Customer".